### PR TITLE
fix(cloud-function): add libs/play as dependency

### PR DESCRIPTION
**Pull request by LeoMcA**
### Problem

Our lint job is failing: https://github.com/mdn/yari/actions/runs/12275409047/job/34250396498

### Solution

Install the missing dependencies

---

## How did you test this change?

In this PR: the lint job succeeds!

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -32,6 +32,10 @@ jobs:
           key: vscode-ripgrep-bin-${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('yarn.lock') }}
           path: node_modules/@vscode/ripgrep/bin/
 
+      - name: Install all npm packages (libs/play)
+        working-directory: libs/play
+        run: npm ci
```
Comment (caugner) [inline .github/workflows/testing.yml:37]: Why is this necessary, given that `yarn install` should also run `npm install` there via the `prepare` script?

https://github.com/mdn/yari/blob/f19459869e3c1d0e1f9210f0bac57da3d1a5d398/package.json#L35-L40

Comment (LeoMcA) [reply .github/workflows/testing.yml]: Ah right, it was an ordering issue: the install happened in `cloud-function` first, before `play`, and so `play` was copied sans `node_modules`: adding `@yari-internal/play` as a dep in `cloud-function` seems to hoist its dependencies into `cloud-functions`' lockfile, avoiding that problem

**Review by caugner — APPROVED** [review]
LGTM, just one nit: IMO the PR title should preferably describe the change / solution , not the problem it solves. Something like: `fix(cloud-Function): add libs/play as dependency`.
